### ISSUE TITLE
[4.0] List of major differences for users moving from J3 to J4

### ISSUE THREAD

**Issue by Scrabble96** [joomla/joomla-cms#26174]
### Is your feature request related to a problem? Please describe.
At the moment, all links to joomla documentation simply go to the documention home page. I hope this will change in due course to the relevant topic.

In the meantime, I am discovering several functions which have completely changed from J3 to J4 and for an everyday, non-techie user, e.g. a site owner who could be a restaurant manager, a church minister,  etc. these changes could be quite bewildering. I'm not talking about differences in core code, which these users shouldn't even be aware of, but user functions like these, for example:

1. In J4 it is now possible to add a child menu item directly under another menu item... including the Home page. In J3, an empty holding menu item (e.g. a separator) with no content is required and a sub-menu item on the Home page is not possible at all.

2. In J4 the option to select the text separator in the breadcrumbs module has disappeared. It is now using the built-in Bootstrap 4 CSS: 
`.breadcrumb-item + .breadcrumb-item::before {
    content: "/"; 
}`

3. Category blog column options have disappeared and are now managed by the addition of css classes.

I am sure there are many more differences.

### Describe the solution you'd like
A clear and precise list of changes in layman's terms to which end-users can refer. I'm sure there will be one. If so, is it in progress and can we view it?

**Comment by ghost** [comment]
Something like https://docs.joomla.org/JDOC:Joomla_4_Tutorials_Project?

**Comment by Sandra97** [comment]
There's none on this subject AFAIK.
@Scrabble96 would it be possible for you to take care of it?

**Comment by Scrabble96** [comment]
> 
> 
> Something like https://docs.joomla.org/JDOC:Joomla_4_Tutorials_Project?

If you're talking about the type of step-by-step instructions with pictures as shown for "Adding menu link in an article", then yes. 

But the type of content that's shown in [https://docs.joomla.org/Potential_backward_compatibility_issues_in_Joomla_4](url), for example, is a long list of technical jargon that is simply gobbledygook to the type of end user I'm talking about. And it doesn't mention anywhere in that long list the three items I mentioned in my first post; at least, not in any language I can understand.

I would suggest keeping end-user guides and developer information in completely separate lists.

**Comment by Scrabble96** [comment]
> 
> 
> There's none on this subject AFAIK.
> @Scrabble96 would it be possible for you to take care of it?

@Sandra97 I just saw your post as I posted mine. Yes, I could start a list and, when sure that nothing further is going to change (per item), write some step-by-step instructions, too. I wouldn't know how to get it onto docs.joomla.org, though. 

There are, however, many aspects of Joomla that I never use and don't anticipate using.

**Comment by Sandra97** [comment]
I will help you with JDocs.
Once your tutorial is done, other users can still add more content.

if you're on Glip, you can contact me there so that I can guide you

**Comment by Scrabble96** [comment]
@Sandra97 I've just looked it up as I've never heard of Glip. If the free version is sufficient, I'll download it later (I'm going out for a while) and contact you then. It's 13:37 here as I write.

**Comment by Scrabble96** [comment]
@Sandra97 and I have opened a new page in JDocs and I - and others - will update it with simple end-user-friendly explanations and instructions. I will close this for now and re-open it when there is more news.
